### PR TITLE
doc improvement for tf.math.asin and tf.math.atan

### PR DESCRIPTION
**Pull request by Sudeepam97**
PR for issue #26492

### CODE REVIEW

File: tensorflow/core/api_def/base_api/api_def_Asin.pbtxt [hunk 1]
```diff
@@ -1,4 +1,19 @@
 op {
   graph_op_name: "Asin"
-  summary: "Computes asin of x element-wise."
+  summary: "Computes the trignometric inverse sine of x element-wise."
+  description: <<END
+The `tf.math.asin` operation returns the inverse of `tf.math.sin`, such that
+if `y = tf.math.sin(x)` then, `x = tf.math.asin(y)`.
```
Comment (martinwicke) [inline tensorflow/core/api_def/base_api/api_def_Asin.pbtxt]: sin isn't invertible, so this is true for the invertible range [-pi/2, pi/2]. That would be worth mentioning.

File: tensorflow/core/api_def/base_api/api_def_Atan.pbtxt [hunk 2]
```diff
@@ -1,4 +1,19 @@
 op {
   graph_op_name: "Atan"
-  summary: "Computes atan of x element-wise."
+  summary: "Computes the trignometric inverse tangent of x element-wise."
+  description: <<END
+The `tf.math.atan` operation returns the inverse of `tf.math.tan`, such that
+if `y = tf.math.tan(x)` then, `x = tf.math.atan(y)`.
```
Comment (martinwicke) [inline tensorflow/core/api_def/base_api/api_def_Atan.pbtxt]: same here

**Review by martinwicke — CHANGES_REQUESTED** [review]
Do we not have acos? 

Assuming we do, the same change needs to be made there as well I assume.

**Comment by Sudeepam97** [comment]
> Do we not have acos?
> 
> Assuming we do, the same change needs to be made there as well I assume.

Actually, I have fixed acos in #26017. Would you be able to look at that PR as well? It has been pending for a long time.

As for the changes, will writing something like...
**"Note: The output of `tf.math.asin` will lie within the invertible range of sine i.e. [-pi/2, pi/2] "**
...be enough or should I add another usage example that demonstrates this?

**Comment by martinwicke** [comment]
I think the wording changes you propose are good. 

This comment also applies to #26017, can you make the same change to cos (except of course the range is different)?

**Comment by Sudeepam97** [comment]
Alright, I've specified the range for all the three operations.

**Review by martinwicke — APPROVED** [review]

**Comment by Sudeepam97** [comment]
@martinwicke will you be able to approve #26017 as well if that PR is fine?

**Pull request merged**